static/js/containers/pages/b2b/B2BReceiptPage.js

```diff
@@ -130,36 +137,68 @@ export class B2BReceiptPage extends React.Component<Props, State> {
-                {title}
-                <span className="description">{readableId}</span>
-              </p>
-              <p className="seats">
-                <span className="description">Seats:</span>
-                {numSeats} (at {formatPrice(itemPrice)} per seat)
-              </p>
-              <p className="email">
-                <span className="description">Email Address:</span>
-                {email}
-              </p>
-              {contractNumber && (
-                <p className="contract-number">
-                  <span className="description">Contract Number:</span>
-                  {contractNumber}
+              <B2BReceiptExplanation className="b2b-receipt-explanation" />
+              <div className="text-box">
+                <h4>
+                  Purchase Summary ({referenceNumber ? referenceNumber : null}):
+                </h4>
+                <p className="order-date">
+                  <span className="description">Order Date:</span>
+                  {moment(createdOn).format("MMMM DD, YYYY")}
                 </p>
-              )}
-              If you encounter any issues please contact{" "}
-              <a href="https://xpro.zendesk.com/hc/requests/new">
-                customer support
-              </a>
-              .
+                {currentUser && currentUser.is_authenticated && (
+                  <p className="customer-name">
+                    <span className="description">Customer Name:</span>
+                    {currentUser.name}
+                  </p>
+                )}
+                <p className="email">
+                  <span className="description">Email Address:</span>
+                  {email}
+                </p>
+                <p className="course-or-program">
+                  <span className="description">Course or program:</span>
+                  {title}
+                  <span className="description">{readableId}</span>
+                </p>
+                <p className="seats">
+                  <span className="description">Seats:</span>
+                  {numSeats} (at {formatPrice(itemPrice)} per seat)
+                </p>
+                {contractNumber && (
+                  <p className="contract-number">
+                    <span className="description">Contract Number:</span>
+                    {contractNumber}
+                  </p>
+                )}
+                {couponCode && (
+                  <p className="coupon_code">
+                    <span className="description">Discount:</span>
+                    {couponCode} Applied
+                  </p>
+                )}
+                {receiptData["card_type"] && receiptData["card_number"] && (
+                  <p className="payment-info">
+                    <span className="description">Payment Method:</span>
+                    {receiptData["card_type"]} | {receiptData["card_number"]}
```

Comment:
Please use dot notation in JS

```suggestion
                {receiptData.card_type && receiptData.card_number && (
                  <p className="payment-info">
                    <span className="description">Payment Method:</span>
                    {receiptData.card_type} | {receiptData.card_number}
```